### ISSUE TITLE
Handle errors more gracefully 

### ISSUE THREAD

**Issue by calum** [calum/alexa-drag-race-skill#12]
At the moment, if a drag queen isn't recognised the user get's no response. 

Request:
```
{
  "session": {
    "new": true,
    "sessionId": "SessionId.433a2169-ec6a-40e4-bcb9-54556429a051",
    "application": {
      "applicationId": "amzn1.ask.skill.fb63dcb7-fdbf-457c-ad1e-614f084a7bfb"
    },
    "attributes": {},
    "user": {
      "userId": "amzn1.ask.account.AFWLHQHHME6UWKTLJR4H6UZZWZBNXIOXVXRKGAT452CANEGOU6CUTNYHOSA2IFKS7XWBDFC6JZNXSZV6FARFEUQKTWKXMLW2JVJJOOME6FWSVJCUJGWSP7RT7JEVC5AXHXDSX4UJ55MDVZJVA7XKTGKLKCLSXZEJKNTZDSRS4TCPEVRKNGIG6I26SCQCTG42JPWFJA4RRYGAVTI"
    }
  },
  "request": {
    "type": "IntentRequest",
    "requestId": "EdwRequestId.8cf30466-c645-42d8-b95e-441430e86574",
    "intent": {
      "name": "getseasonfromqueen",
      "slots": {
        "queen": {
          "name": "queen",
          "value": "is thorgy Thor"
        }
      }
    },
    "locale": "en-GB",
    "timestamp": "2017-10-28T23:13:43Z"
  },
  "context": {
    "AudioPlayer": {
      "playerActivity": "IDLE"
    },
    "System": {
      "application": {
        "applicationId": "amzn1.ask.skill.fb63dcb7-fdbf-457c-ad1e-614f084a7bfb"
      },
      "user": {
        "userId": "amzn1.ask.account.AFWLHQHHME6UWKTLJR4H6UZZWZBNXIOXVXRKGAT452CANEGOU6CUTNYHOSA2IFKS7XWBDFC6JZNXSZV6FARFEUQKTWKXMLW2JVJJOOME6FWSVJCUJGWSP7RT7JEVC5AXHXDSX4UJ55MDVZJVA7XKTGKLKCLSXZEJKNTZDSRS4TCPEVRKNGIG6I26SCQCTG42JPWFJA4RRYGAVTI"
      },
      "device": {
        "supportedInterfaces": {}
      }
    }
  },
  "version": "1.0"
}
```

Response:
```
The remote endpoint could not be called, or the response it returned was invalid.
```

Error message in logs:
```
2017-10-28T23:13:45.566Z a46aca2d-bc35-11e7-9291-77f9ca161688 TypeError: Cannot read property 'original' of undefined
at /var/task/src/drag_race/api.js:37:46
at Request.request.get [as _callback] (/var/task/src/drag_race/api.js:14:12)
at Request.self.callback (/var/task/node_modules/request/request.js:186:22)
at emitTwo (events.js:106:13)
at Request.emit (events.js:191:7)
at Request.<anonymous> (/var/task/node_modules/request/request.js:1163:10)
at emitOne (events.js:96:13)
at Request.emit (events.js:188:7)
at IncomingMessage.<anonymous> (/var/task/node_modules/request/request.js:1085:12)
at IncomingMessage.g (events.js:292:16)
```

**Comment by calum** [comment]
Working on this now

**Comment by calum** [comment]
closed by #20
